app/lib/bk/compat/parsers/bitrise/steps.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'active_support//core_ext/object/blank'
+
+module BK
+  module Compat
+    module BitriseSteps
+      # Implementation of Bitrise step translations
+      class Translator
+        VALID_STEP_TYPES = %w[change-workdir git-clone script].freeze
+
+        def matcher(type, _inputs)
+          VALID_STEP_TYPES.include?(type.downcase)
+        end
+
+        def translator(type, inputs)
+          send("translate_#{type.downcase.gsub('-', '_')}", inputs)
+        end
+
+        def translate_change_workdir(inputs)
+          [
+            generate_change_workdir_command(inputs['path'], inputs['is_create_path'])
+          ]
+        end
+
+        def generate_change_workdir_command(path, is_create_path)
+          if is_create_path && path.present?
+            "mkdir #{path} && cd #{path}"
+          elsif !is_create_path && path.present?
+            "cd #{path}"
+          else
+            '# Invalid change-workdir step configuration!'
+          end
+        end
```

Comment:
being that `path` is mandatory I would refactor this two as:

```suggestion
        def translate_change_workdir(inputs)
          return '# Invalid change-workdir step configuration!' unless inputs.include?('path')
          
          [
            inputs['is_create_path'] ? "mkdir #{inputs['path']}" : nil,
            "cd #{inputs['path']}"
          ].compact
        end
```

Reply:
Note that this change would also avoid having to add `activesupport` as a dependency at all

Reply:
Changed - and I also removed `activesupport` entirely in the latest commit (I'll still need to alter the behaviour in the other incoming parser steps in #147)